ports/samd/boards/GENERIC_SAMD51P19/board.json

```diff
@@ -0,0 +1,15 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "USB"
+    ],
+    "mcu": "samd51",
+    "product": "Generic SAM51P19",
+    "variants": {
+        "WLAN": "WiFi support using a ESP32 module with NINA firmware"
```

Comment:
Looks like this board does not have a variant defined?

Reply:
Removed